docs/sql/index.md

```diff
@@ -1,407 +1,391 @@
+- [SELECT](#select)
+- [INSERT](#insert)
+- [DELETE](#delete)
+- [UPDATE](#update)
+- [SET](#set)
+- [SHOW](#show)
 
-#### Logical expressions
+### SELECT
 
-Any expression which returns a boolean, i.e. `true` or `false`, is a logical expression.
-
-| Example          | Description                                                  |
-| ---------------- | ------------------------------------------------------------ |
-| `1 > 2`          | Integer comparison.                                          |
-| `1 + 2 == 3`     | Equality comparison between a constant and a computed value. |
-| `true AND false` | Boolean and.                                                 |
-| `true OR false`  | Boolean or.                                                  |
-| `NOT true`       | Boolean inverse.                                             |
-
-#### Function calls
-
-| Example         | Description                                        |
-| --------------- | -------------------------------------------------- |
-| `lower('JOHN')` | Apply the function `lower` to the string `'JOHN'`. |
-
-#### Table identifiers
-
-| Example       | Description               |
-| ------------- | ------------------------- |
-| `inventory`   | Refers to a table.        |
-| `"inventory"` | Refers to the same table. |
-
-#### Column references
-
-| Example                    | Description                                             |
-| -------------------------- | ------------------------------------------------------- |
-| `inventory_id`             | Refers to a column.                                     |
-| `"inventory_id"`           | Refers to the same column.                              |
-| `"inventory.inventory_id"` | Refers to the same column, explicitly naming its table. |
-
-#### Wildcards
-
-Special "star" expressions which select all the columns of a table.
-
-| Example       | Description                                             |
-| ------------- | ------------------------------------------------------- |
-| `*`           | Refers to all columns of a table identified by context. |
-| `inventory.*` | Refers to all columns of the `inventory` table.         |
-
-#### Parenthesized expressions
-
-Sub-expressions can be enclosed in parentheses for grouping and to override operator precedence.
-
-| Example       | Description             |
-| ------------- | ----------------------- |
-| `1 + (2 / 3)` | One plus a fraction.    |
-| `(1 + 2) / 3` | A sum divided by three. |
-
-### `CREATE TABLE`
-
-A `CREATE TABLE` statement creates a new, initially empty table in the database.
+```ebnf
+SELECT projection FROM relation [ WHERE predicate ]
+```
 
-The syntax of the `CREATE TABLE` statement is:
+The query languge is a strict superset of the subscription language.
+The main differences are seen in column projections and joins.
 
-> **CREATE TABLE** _table_name_ (_column_name_ _data_type_, ...);
+The subscription api only supports wildcard projections,
```

Comment:
If we're going to talk about "wildcard projections" this much, I think we should define that term somewhere. It would also be nice to say *why* subscriptions only support wildcard queries.